pom.xml

```diff
@@ -909,7 +909,7 @@
     <snappy.version>1.1.10.4</snappy.version>
     <xz.version>1.9</xz.version>
     <zstd-jni.version>1.5.5-2</zstd-jni.version>
-    <hbase-thirdparty.version>4.1.5</hbase-thirdparty.version>
+    <hbase-thirdparty.version>4.1.6-28374-d2c399f-SNAPSHOT</hbase-thirdparty.version>
```

Comment:
We also need to sync the internal protobuf version same as hbase-thirdparty.

Reply:
Ah, so it seems from [HBASE-27578](https://issues.apache.org/jira/browse/HBASE-27578). Are there any other related dependencies?

cc @Apache9

Reply:
Yes we do same for Jackson.

Reply:
Sure but I think we usually bump all jackson related dependenies which are jackson-core, jackson-annotations and jackson-databind with thirdparty bump and keep it in sync with jaxrs json. 

Ref: https://github.com/apache/hbase/commit/a0fdfa782b7ef18c1d0459dcb46ae38feb489b67

Reply:
I was trying to bump jackson in our internal fork. Seems there are more changes needed as jackson-core fails in aggregate step.
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:3.4.1:shade (aggregate-into-a-jar-with-relocated-third-parties) on project hbase-shaded-client: Error creating shaded jar: Problem shading JAR /home/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.16.1/jackson-core-2.16.1.jar entry META-INF/versions/21/com/fasterxml/jackson/core/io/doubleparser/FastDoubleSwar.class: java.lang.IllegalArgumentException: Unsupported class file major version 65 -> [Help 1]
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:3.4.1:shade (aggregate-into-a-jar-with-relocated-third-parties) on project hbase-shaded-mapreduce: Error creating shaded jar: Problem shading JAR /home/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.16.1/jackson-core-2.16.1.jar entry META-INF/versions/21/com/fasterxml/jackson/core/io/doubleparser/FastDoubleSwar.class: java.lang.IllegalArgumentException: Unsupported class file major version 65 -> [Help 1]
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:3.4.1:shade (aggregate-into-a-jar-with-relocated-third-parties) on project hbase-shaded-testing-util: Error creating shaded jar: Problem shading JAR /home/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.16.1/jackson-core-2.16.1.jar entry META-INF/versions/21/com/fasterxml/jackson/core/io/doubleparser/FastDoubleSwar.class: java.lang.IllegalArgumentException: Unsupported class file major version 65 -> [Help 1]
```
We can go ahead with this PR as is. Let me handle this with another JIRA.

Reply:
https://issues.apache.org/jira/browse/HBASE-28392

Reply:
Raised draft at https://github.com/apache/hbase/pull/5698

Reply:
The build will fail with simple bump. do you mind taking a look at https://github.com/apache/hbase/pull/5698? Please feel free to pull in changes from #5698 to this PR.